benchmark/micro/window/window_mode_variable_100.benchmark

```diff
@@ -1,23 +0,0 @@
-# name: benchmark/micro/window/window_mode_variable_100.benchmark
-# description: Moving mode performance, variable 100 element frame
-# group: [window]
-
-name Windowed MODE, Variable 100
-group window
-
-load
-create table rank100 as
-    select b % 100 as a, b from range(10000000) tbl(b)
-
-run
-select sum(m)
-from (
-    select mode(a) over (
-        order by b asc
-        rows between mod(b * 47, 521) preceding and 100 - mod(b * 47, 521) following) as m
-    from rank100
-    ) q;
-
-result I
-494986272
```

Comment:
Looking at the values within the windows they are all unique, hence `mode` is not defined (i.e. if every value occurs once, every value is the "mode"). I think it's a good idea to remove this benchmark and perhaps replace it with a consistent one.